File: app/src/main/java/com/mrboomdev/awery/data/db/item/DBRepository.java
package com.mrboomdev.awery.data.db.item;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "repository")
public class DBRepository {
	@PrimaryKey
	@NonNull
	public String url;
	@NonNull
	public String manager;
	@ColumnInfo(name = "is_enabled", defaultValue = "true")
	public boolean isEnabled = true;

	public DBRepository(@NonNull String url, @NonNull String manager) {
		this.url = url;
		this.manager = manager;
	}
}

File: .old-shared/src/commonMain/kotlin/com/mrboomdev/awery/data/entity/DBRepository.kt
package com.mrboomdev.awery.data.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class DBRepository(
	@PrimaryKey val url: String,
	val manager: String,
	val isEnabled: Boolean = true,
	val title: String? = null,
	val description: String? = null,
	val author: String? = null
)